src/pages/Landing.tsx

```diff
@@ -26,6 +28,10 @@ export const Landing = () => {
   const location = useLocation();
   const history = useHistory();
 
+  useEffect(() => {
+    emitMetric(METRIC_NAMES.viewHome);
+  }, []);
+
```

Comment:
Carl created a custom middleware in Freighter that automatically emits an event on routing. Might be nice to do something similar in a future iteration